modules/java_api/samples/java_api_2.0_sample/Main.java

```diff
@@ -0,0 +1,113 @@
+import org.intel.openvino.*;
+import org.intel.openvino.Core;
+import org.opencv.core.*;
+import org.opencv.highgui.HighGui;
+import org.opencv.imgcodecs.*;
+import org.opencv.imgproc.Imgproc;
```

Comment:
I'd like recommend replace deprecated sample completely. It's OK to keep both APIs in the tests but samples should propose the latest one.

Reply:
I agree, but I think we should first add async methods for Java API 2.0.

Reply:
These are two different demos so why not upgrade a synchronous sample first?